pkg/kube/inject/webhook.go

```diff
@@ -610,7 +609,7 @@ func adjustInitContainerUser(finalPod *corev1.Pod, originalPod *corev1.Pod, prox
 	tproxy := false
 	if proxyConfig.InterceptionMode == meshconfig.ProxyConfig_TPROXY {
 		tproxy = true
-	} else if mode, found := finalPod.Annotations[annotation.SidecarInterceptionMode.Name]; found && mode == iptablesconstants.TPROXY {
+	} else if mode, found := finalPod.Annotations[annotation.SidecarInterceptionMode.Name]; found && mode == "TPROXY" {
```

Comment:
Don't care much to debate it if you prefer this, but aside from this constant removal (which is good), the rest I think are a slight detriment. A bit easier to make typos and a bit harder to find where we use specific tables